install/resources/upgrade50.php

```diff
@@ -35,3 +35,22 @@ function upgrade50_dbchanges()
 	$output->print_contents("<p>Click next to continue with the upgrade process.</p>");
 	$output->print_footer("50_done");
 }
+
+function upgrade50_verify_adminemail()
+{
+	global $output, $cache, $db, $mybb;
+
+	$output->print_header("Verifying Admin Email");
+	if(empty($mybb->settings['adminemail']))
+	{
+		echo "<p>Updating admin email settings...</p>";
+		echo "<p><small>P.D: Field can not be empty</small></p>";
+		$db->update_query('settings', array('value' => $mybb->user['email']), "name='adminemail'");
+	}
+	else
+	{
+		echo "<p>Admin email verified success...</p>";		
+	}
+	$output->print_contents("<p>Click next to continue with the upgrade process.</p>");
+	$output->print_footer("50_done");
```

Comment:
```suggestion
	$output->print_footer("50_update_adminemail");
```

<-- then change this to `$output->print_footer("50_update_adminemail");` and add a `upgrade50_update_adminemail`, something along the lines of... -->